corehq/apps/users/views/__init__.py

```diff
@@ -704,10 +705,15 @@ def create_domain_permission_mirror(request, domain):
         mirror_domain_name = form.cleaned_data.get("mirror_domain")
         mirror_domain = Domain.get_by_name(mirror_domain_name)
         if mirror_domain is not None:
-            mirror = DomainPermissionsMirror(source=domain, mirror=mirror_domain_name)
-            mirror.save()
-            message = _('You have successfully added the project space "{mirror_domain_name}".')
-            messages.success(request, message.format(mirror_domain_name=mirror_domain_name))
+            try:
+                mirror = DomainPermissionsMirror(source=domain, mirror=mirror_domain_name)
+                mirror.save()
+            except IntegrityError:
+                message = _(mirror_domain_name + ' has already been added.')
```

Comment:
This needs to use `format` like the error message below. If you pass a dynamic string to `_`, the calculated string isn't going to have a translation, so it won't get translated. [More about translations](https://commcare-hq.readthedocs.io/translations.html).

Reply:
Understood, just updated it to use format.

Reply:
I don't see a change - possibly forgot to push changes?

Reply:
Sorry, yeah I was looking at Ethan's notes and forgot to push. Should be up now.